File: weekly_test/total_bill.js
function total_bill(bill_list) {

  var total = 0;
  for(var i = 0;i < bill_list.length;i ++){
   //  console.log(bill_list[i]);
     for(var j = 0;j < bill_list[i].length;j ++){
        var price = bill_list[i][j]['price'].slice(1);
      //  console.log(price);
        total = +total + +price;
     }
  }
  
  //console.log(total);
  return total;
}

module.exports = total_bill;

File: weekly_test/total_bill.pl
#!/usr/bin/perl -w

die "Usage: $0 <file>\n" if @ARGV != 1;

$filename = $ARGV[0];
my $total = 0.00;
open my $f,$filename or die "Can not open $filename\n";

while ($line = <$f>){
  #  print $line;
     if($line =~ m/price/){
       my @price = split ':',$line;
       my $temp = $price[2] =~ s/,//gr;
       $temp = $temp =~ s/"//gr;
       $temp = $temp =~ s/\$//gr;
       $temp = $temp =~ s/}//gr;
       $temp = $temp =~ s/\s+//gr;
       $total += $temp;
      # print "$temp ";
     }

}

printf "\$%.2f\n",$total;
